tests/integration/test_match_process_uid_against_data_owner/test.py

```diff
@@ -33,7 +33,7 @@ def test_different_user(started_cluster):
         node.start_clickhouse(start_wait_sec=3)
 
     log = node.grep_in_log("Effective")
-    expected_message = "Effective user of the process (.*) does not match the owner of the data (.*)\\. Run under 'sudo -u .*'\\."
+    expected_message = "Effective user of the process \\(.+?\\) does not match the owner of the data \\(.+?\\)\\. Run under 'sudo -u .*'\\."
```

Comment:
@alexey-milovidov It will not work with escaping:
```
>>> import re
>>> re.search('is (\\d+)x faster', 'ClickHouse is 1000x faster than your database').groups()
('1000',)
>>> print(re.search('is \\(\\d+\\)x faster', 'ClickHouse is 1000x faster than your database'))
None
```

Reply:
But actually the message really contains brackets: https://github.com/ClickHouse/ClickHouse/blob/4588e146a63dc9cbe79d4a81980886b35463c806/programs/server/Server.cpp#L845